bika/lims/skins/bika/portlet_accreditation.pt

```diff
@@ -7,7 +7,9 @@
 <div metal:define-macro="portlet"
      tal:define="portal_state context/@@plone_portal_state;
                  portal_url portal_state/portal_url;
-                 plone_view context/@@plone;">
+                 anonymous portal_state/anonymous;
+                 plone_view context/@@plone;"
+     tal:condition="python: not anonymous">
```

Comment:
You could even skip the `python` part in the condition and just write `tal:condition="not:anonymous"`. But this is also fine;) well done